users/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls    import path
-from users.views    import SignupView
+from users.views    import UserInformationView
 
 
 
 urlpatterns = [
-            path('/signup', SignupView.as_view())
+            path('/userinfo', UserInformationView.as_view()),
```

Comment:
```suggestion
            path('/information', UserInformationView.as_view()),
```
main url.py에 user가 이미 있기 때문에 다음과 같이 수정하면 좋을 것 같습니다.